VERSION.txt

```diff
@@ -1 +1 @@
-15.0.1
+15.1.0
```

Comment:
I might argue that this is a bugfix and so the new version should be 15.0.2.  But it really doesn't matter.